web/client/product/plugins/Fork.jsx

```diff
@@ -9,8 +9,8 @@ const React = require('react');
 const Fork = React.createClass({
     render() {
         return (
-            <a href="https://github.com/geosolutions-it/MapStore2" className="ms-fork-button">
-                <img style={{position: "absolute", top: 55, right: 0, border: 0, zIndex: 100}} src="https://camo.githubusercontent.com/e7bbb0521b397edbd5fe43e7f760759336b5e05f/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f6769746875622f726962626f6e732f666f726b6d655f72696768745f677265656e5f3030373230302e706e67" alt="Fork me on GitHub" dataCanonicalSrc="https://s3.amazonaws.com/github/ribbons/forkme_right_green_007200.png"/>
+            <a href="https://github.com/geosolutions-it/MapStore2" >
+                <img className="ms-fork-button" src="https://camo.githubusercontent.com/e7bbb0521b397edbd5fe43e7f760759336b5e05f/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f6769746875622f726962626f6e732f666f726b6d655f72696768745f677265656e5f3030373230302e706e67" alt="Fork me on GitHub" dataCanonicalSrc="https://s3.amazonaws.com/github/ribbons/forkme_right_green_007200.png"/>
```

Comment:
This is the image for the upper right corner: you can find the ribbons also for the upper left corner here: https://github.com/blog/273-github-ribbons